test/fuzz/luaL_loadbuffer/serializer.cc

```diff
@@ -235,15 +235,16 @@ class Context {
 	}
```

Comment:
commit message: it is worth to explain why do you add these checks.
You say this in PR description, I would add stats to commit message too:
> Skipped break without cycle solved 11150 (22%) errors, skipped return without function solved 599 (1%) errors

Reply:
Fixed.